Archipelago.HollowKnight/Archipelago.HollowKnight.csproj

```diff
@@ -49,52 +49,52 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="Archipelago.MultiClient.Net">
-      <HintPath>..\DLLs\Archipelago.MultiClient.Net.dll</HintPath>
+      <HintPath>$(HollowKnightRefs)/Mods/Archipelago.HollowKnight/Archipelago.MultiClient.Net.dll</HintPath>
     </Reference>
     <Reference Include="Assembly-CSharp">
-      <HintPath>..\DLLs\Assembly-CSharp.dll</HintPath>
+      <HintPath>$(HollowKnightRefs)/Assembly-CSharp.dll</HintPath>
     </Reference>
     <Reference Include="Assembly-CSharp-firstpass">
-      <HintPath>..\DLLs\Assembly-CSharp-firstpass.dll</HintPath>
+      <HintPath>$(HollowKnightRefs)/Assembly-CSharp-firstpass.dll</HintPath>
     </Reference>
     <Reference Include="GalaxyCSharp">
-      <HintPath>..\DLLs\GalaxyCSharp.dll</HintPath>
+      <HintPath>$(HollowKnightRefs)/GalaxyCSharp.dll</HintPath>
     </Reference>
     <Reference Include="ItemChanger">
-      <HintPath>..\DLLs\ItemChanger.dll</HintPath>
+      <HintPath>$(HollowKnightRefs)/Mods/ItemChanger/ItemChanger.dll</HintPath>
     </Reference>
     <Reference Include="MenuChanger">
-      <HintPath>..\DLLs\MenuChanger.dll</HintPath>
+      <HintPath>$(HollowKnightRefs)/Mods/MenuChanger/MenuChanger.dll</HintPath>
     </Reference>
     <Reference Include="MMHOOK_Assembly-CSharp">
-      <HintPath>..\DLLs\MMHOOK_Assembly-CSharp.dll</HintPath>
+      <HintPath>$(HollowKnightRefs)/MMHOOK_Assembly-CSharp.dll</HintPath>
     </Reference>
     <Reference Include="MMHOOK_PlayMaker">
-      <HintPath>..\DLLs\MMHOOK_PlayMaker.dll</HintPath>
+      <HintPath>$(HollowKnightRefs)/MMHOOK_PlayMaker.dll</HintPath>
     </Reference>
     <Reference Include="Mono.Cecil">
-      <HintPath>..\DLLs\Mono.Cecil.dll</HintPath>
+      <HintPath>$(HollowKnightRefs)/Mono.Cecil.dll</HintPath>
     </Reference>
     <Reference Include="Mono.Security">
-      <HintPath>..\DLLs\Mono.Security.dll</HintPath>
+      <HintPath>$(HollowKnightRefs)/Mono.Security.dll</HintPath>
     </Reference>
     <Reference Include="MonoMod.RuntimeDetour">
-      <HintPath>..\DLLs\MonoMod.RuntimeDetour.dll</HintPath>
+      <HintPath>$(HollowKnightRefs)/MonoMod.RuntimeDetour.dll</HintPath>
     </Reference>
     <Reference Include="MonoMod.Utils">
-      <HintPath>..\DLLs\MonoMod.Utils.dll</HintPath>
+      <HintPath>$(HollowKnightRefs)/MonoMod.Utils.dll</HintPath>
     </Reference>
     <Reference Include="netstandard">
       <HintPath>$(HollowKnightRefs)/netstandard.dll</HintPath>
     </Reference>
     <Reference Include="Newtonsoft.Json">
-      <HintPath>..\DLLs\Newtonsoft.Json.dll</HintPath>
+      <HintPath>$(HollowKnightRefs)/Newtonsoft.Json.dll</HintPath>
```

Comment:
Please implement #29 for this DLL. I want to use the version shipped with multiclient, not the one used by moddingapi.